jobs/index.html

```diff
@@ -26,5 +26,10 @@ <h3 id="benefits">Benefits</h3>
         </ul>
         <p>If something comes up for a team member that is not covered by one of these benefits, we will do our best to establish a new benefit to help them.</p>
         <p>Read more about our job philosophy <a href="/blog/2015/04/23/work-sustainably">here</a>.</p>
+
+        <h3 id="faq">FAQ</h3>
+        <p><strong>Is it really a 20-hour work week?</strong><br />Yes.  We only work 20 hours a week, whether it's billable or non-billable work.</p>
+        <p><strong>How much do you pay?</strong><br />We consider this a full-time job, so we offer a full-time salary and benefits.  We don't try to compete with Amazon or Facebook on salary, but we're probably at or above the median for software developers as a whole.</p>
```

Comment:
I'd go with "We almost certainly pay above the US Median for software developers"